src/Bundler.js

```diff
@@ -334,6 +459,12 @@ module.exports = class Bundler {
     });
 
     archive.append(JSON.stringify(packageJson, null, '  '), { name: 'package.json' });
+    archive.append(`
+account_id = "b4adf6cfdac0918eb6aa5ad033da0747"
+name = "helix-universal-test"
```

Comment:
this is very specific... does this need to be here?

Reply:
Fixed in https://github.com/adobe/helix-deploy/pull/324/commits/3a11aa1602b4766c7327029a521f03068ef2824e